.eslintrc.js

```diff
@@ -13,8 +13,15 @@ module.exports = {
   extends: ['modular-app'],
   rules: {
     strict: [WARN, 'global'],
+    'import/no-extraneous-dependencies': ['error', { devDependencies: false }],
```

Comment:
I think adding `devDependencies` is fine for the use case outlined in my other comment.

Reply:
I'd prefer to use inline exceptions to the rule rather than allowing `devDependencies` really - enabling `devDependencies` here wouldn't prevent the error the PR is intended to resolve...